static/public/index.html

```diff
@@ -11250,6 +11250,1176 @@ <h1 class="text-hero-large-bold mb-4">Styles :</h1>
+          Success</label>
+
+        <div
+          class="w-80 px-2 py-2 border-2 border-solid rounded-md border-success-shd5 hover:border-success-shd3 relative text-white focus-within:text-success">
+          <div class="absolute inset-y-0 ml-2 flex items-center">
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path
+                d="M0 4C0 1.79086 1.79086 0 4 0H20C22.2091 0 24 1.79086 24 4V20C24 22.2091 22.2091 24 20 24H4C1.79086 24 0 22.2091 0 20V4Z"
+                fill="#F2F9FF" />
+              <path
+                d="M14.085 14.4023H9.91504L8.97852 17H7.625L11.4258 7.04688H12.5742L16.3818 17H15.0352L14.085 14.4023ZM10.3115 13.3223H13.6953L12 8.66699L10.3115 13.3223Z"
+                fill="#8D9A9E" />
+            </svg>
+          </div>
+          <input type="text" id="success-six"
+            class="w-10/12 text-grey placeholder-grey-shd5 focus:outline-none focus:placeholder-grey ml-12"
+            placeholder="Enter text">
+          <div class="absolute inset-y-0 right-0 mx-2 flex items-center">
+            <svg class="fill-current cursor-pointer" width="11" height="9" viewBox="0 0 11 9" fill="none"
+              xmlns="http://www.w3.org/2000/svg">
+              <path
+                d="M3.66681 6.39061L1.47148 4.19527L0.528809 5.13794L3.66681 8.27594L10.1381 1.8046L9.19547 0.861938L3.66681 6.39061Z" />
+            </svg>
+          </div>
+        </div>
+      </div>
+
+      <!-- disabled input -->
+      <div class="disabled-wrapper mx-auto">
+        <label class="inline-block mb-2 bg-grey-shd7 px-2 py-2 rounded-md cursor-not-allowed">Label
+          Disabled</label>
+        <div class="w-80 px-2 py-2 border-2 border-solid rounded-md border-grey-shd7 relative disabled">
+          <div class="absolute inset-y-0 ml-2 flex items-center">
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path
+                d="M0 4C0 1.79086 1.79086 0 4 0H20C22.2091 0 24 1.79086 24 4V20C24 22.2091 22.2091 24 20 24H4C1.79086 24 0 22.2091 0 20V4Z"
+                fill="white" />
+              <path
+                d="M14.085 14.4023H9.91504L8.97852 17H7.625L11.4258 7.04688H12.5742L16.3818 17H15.0352L14.085 14.4023ZM10.3115 13.3223H13.6953L12 8.66699L10.3115 13.3223Z"
+                fill="#C5DCE3" />
+            </svg>
+          </div>
+          <input type="text"
+            class="w-11/12 cursor-not-allowed placeholder-grey-shd5 focus:outline-none focus:placeholder-grey ml-8"
+            placeholder="Enter text">
+        </div>
+      </div>
+    </div>
+
+    <!--  Horizontal Sliders -->
+
+    <div class="grid grid-cols-3 gap-4 pl-12 py-12 mt-16 mx-10 border-2 border-dotted border-accent">
+      <div class="flex flex-col items-left">
+        <h4>Start</h4>
```

Comment:
I think container for start progress is not like design

Reply:
could you please snapshot it?